src/caretogether-pwa/src/Families/ManageUserDrawer.tsx

```diff
@@ -51,18 +64,33 @@ export function ManageUserDrawer({
 
   const withBackdrop = useBackdrop();
 
-  async function invitePersonUser() {
-    await withBackdrop(async () => {
-      const inviteLink = await api.users.generatePersonInviteLink(
-        organizationId,
-        locationId,
-        adult.id
-      );
-      await navigator.clipboard.writeText(inviteLink);
-      alert(
-        `The invite link for ${personNameString(adult)} has been copied to your clipboard.`
-      );
+  const [inviteLink, setInviteLink] = useState<string | null>(null);
+  const [inviteLinkLoading, setInviteLinkLoading] = useState(false);
+
+  const [noticeOpen, setNoticeOpen] = useState(false);
+
+  async function generateInviteLink() {
+    setInviteLinkLoading(true);
+
+    await new Promise((resolve) => {
```

Comment:
Pretty sure I did this just to show the loading state on the video and forgot to remove 😂 
Will remove it

Reply:
I just pushed a proposed change that might handle this better using a delayed-start animation; let me know what you think and if I've missed something!

Reply:
Haha! I didn't see your message and figured it was because there was a long delay on the server. 😁 I'll go ahead and merge with my animation tweak included since it shouldn't hurt anything, but most likely no one will ever see it. 😂